package.json

```diff
@@ -35,10 +35,12 @@
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-jshint": "^1.0.0",
+    "ember-cli-legacy-blueprints": "0.1.1",
```

Comment:
If necessary, we could add conditional installation of this dependency to the `ember-engines` installation blueprint (i.e., only for ember-cli >= 2.6), but this could cause some tricky unexpected failures as developers upgrade from 2.5 --> 2.6 and don't understand why their in-repo-engine generator started to fail